### PR TITLE
Adding tests

### PR DESCRIPTION
**Pull request by atarkowska**
This PR clean up readme to help deploy application and adds test

### CODE REVIEW

**Comment by atarkowska** [comment]
And that is green as well :-)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+env: PIP_ALLOW_EXTERNAL=PIL PIP_ALLOW_UNVERIFIED=PIL
+
+before_script:
+  - curl -O http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz
+  - curl -O http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz
+  - gzip -d GeoIP.dat.gz
+  - gzip -d GeoLiteCity.dat.gz
+  - mv Geo* omero_qa
+
+script: python manage.py test registry --settings=omero_qa.settings-test
```
Comment (sbesson) [inline .travis.yml]: @aleksandra-tarkowska: is there a way to pass a verbose flag to list passed/failed tests in Travis?

Comment (atarkowska) [inline .travis.yml]: `-v 2` gives 

```
test_bad_agent (omero_qa.registry.tests.RegistryTestCase) ... ok
test_current_version (omero_qa.registry.tests.RegistryTestCase) ... ERROR
test_empty_hit (omero_qa.registry.tests.RegistryTestCase) ... ok
test_old_version (omero_qa.registry.tests.RegistryTestCase) ... ERROR
======================================================================
ERROR: test_current_version (omero_qa.registry.tests.RegistryTestCase)
...
```

Comment (sbesson) [inline .travis.yml]: I think it's worth pushing Want a commit for that?

**Comment by sbesson** [comment]
Sorry @aleksandra-tarkowska. Did not see the last commit pushed. Looks good to me. Merging

**Pull request merged**